mergify_engine/subscription.py

```diff
@@ -115,22 +116,45 @@ def to_dict(self) -> SubscriptionDict:
             "features": [f.value for f in self.features],
         }
 
+    RETENTION_SECONDS = 60 * 60 * 24 * 3  # 3 days
+    VALIDITY_SECONDS = 3600
+
+    async def _have_expired(self) -> bool:
+        expire_at: int = await self.redis.ttl(
```

Comment:
```suggestion
        ttl: int = await self.redis.ttl(
```
expire_at implies it's a timestamp and it's not